execute/row_fn.go

```diff
@@ -206,7 +214,7 @@ func NewTablePredicateFn(fn *semantic.FunctionExpression, scope compiler.Scope)
 }
 
 func (f *TablePredicateFn) Prepare(tbl flux.Table) (*TablePredicatePreparedFn, error) {
-	fn, err := f.prepare(tbl.Key().Cols(), nil)
+	fn, err := f.prepare(tbl.Key().Cols(), nil, false)
```

Comment:
I'm confused about why we're passing bool literals as arguments to `prepare`. I thought we would be checking whether the function expression has a non-nil value for the `Vectorized` field, and setting the argument for `vectorized` based on that check.

Reply:
This is after the check for vectorization. We've now either passed the vectorized function or not. It's mostly because, if it's vectorized, then the input columns need to be wrapped in vector types. Rather than expose that code to the map transformation, I chose to just add a boolean to this private helper struct.